ucm2/codecs/wcd938x/AnalogMic1EnableSeq.conf

```diff
@@ -0,0 +1,7 @@
+# Analog Microphone on Input1 via ADC1
+EnableSequence [
+	cset "name='ADC1_MIXER Switch' 1"
+	cset "name='ADC1 Switch' 1"
+	cset "name='ADC1 Volume' 10"
```

Comment:
Volume presets should be only in `BootSequence`. See `Device Volume` section in [this document](https://www.alsa-project.org/alsa-doc/alsa-lib/group__ucm__conf.html).

Reply:
Ack, fixed thanks